main.py

```diff
@@ -169,56 +173,50 @@ async def check(ctx: commands.Context):
                 await member.add_roles(
                     role
                 )
-                await ctx.send(f"Fixed team roles for {member.display_name} - now on {account.team.name}")
+                await ctx.send(
+                    f'Fixed team roles for {member.display_name} - now on '
+                    f'{account.team.name}.'
+                )
         await ctx.send('Done')
 
 
 @bot.command()
 @commands.is_owner()
 async def restart(ctx: commands.Context, *, _: str = None):
-    await ctx.send(f'Shutting down server & Polympics client...')
+    await ctx.send('Shutting down server & Polympics client...')
     await server.shutdown()
     await server.cleanup()
     await polympics_client.close()
-    await ctx.send(f'Complete. Shutting down bot.')
+    await ctx.send('Complete. Shutting down bot.')
     sys.exit(0)
 
 
 @bot.event
 async def on_user_update(before: discord.User, after: discord.User):
     if (account := await polympics_client.get_account(before.id)) is not None:
-        
-        if before.avatar != after.avatar:
-            ext = 'gif' if after.is_avatar_animated() else 'png'
-            avatar_url = f'https://cdn.discordapp.com/avatars/{account.id}/{after.avatar}.{ext}'
-        else:
-            avatar_url = account.avatar_url
-        
         await polympics_client.update_account(
-            account, name=after.name, discriminator=after.discriminator, avatar_url=avatar_url
+            account,
+            name=after.name,
+            discriminator=after.discriminator,
+            avatar_url=str(after.avatar_url)
```

Comment:
I wasn't using avatar_url explicitly, because it returns the avatar sized.

Reply:
Good point. The issue with your implementation was that it didn't handle default avatars.

Reply:
```suggestion
            avatar_url=str(after.avatar_url).split('?')[0]
```
Bit hacky but how's this?

Reply:
Looks good.